pimd/pimd.h

```diff
@@ -90,10 +90,10 @@
 #define PIM_MASK_PIM_PACKETDUMP_RECV (1 << 4)
 #define PIM_MASK_PIM_TRACE           (1 << 5)
 #define PIM_MASK_PIM_TRACE_DETAIL    (1 << 6)
-#define PIM_MASK_IGMP_EVENTS         (1 << 7)
-#define PIM_MASK_IGMP_PACKETS        (1 << 8)
-#define PIM_MASK_IGMP_TRACE          (1 << 9)
-#define PIM_MASK_IGMP_TRACE_DETAIL   (1 << 10)
+#define PIM_MASK_GM_EVENTS (1 << 7)
```

Comment:
Can you align or fix all the indentation with a separate commit?

Reply:
Fixed, Thanks.

Reply:
@ton31337 : The CI warnings are for this alignment. Since this comment is fixed, CI has thrown warning, we can ignore it.